test/features/helpers/files.rb

```diff
@@ -0,0 +1,11 @@
+def get_asset(filename, pdf = false)
+  entries = Dir.entries('./features/helpers/resources')
+  file_name = entries
+    .select {|f|
+      pdf ?
+      f.end_with?('.pdf') :
+      !f.end_with?('.pdf')
+    }
+    .detect {|f| f.include? filename}
```

Comment:
This can return `nil` right? Is that ever OK?

Reply:
this is copy paste from monster with trimming of dead code, so I'm not fully aware of how it works to be honest.
By returning nil you mean that `file_name` can be `nil`, right? If so, that's okay, because it just means the test fails

Reply:
Yes that's what I mean.

I guess the test would fail, but the failure reason might not be obvious. Not a blocker so happy to leave it as is

Reply:
I will leave as is, I'm trying to limit too much refactoring